code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2288,10 +2288,14 @@
 
 	if(..())
 		return 1
-
+		
+	if(M.species.name=="Plasmaman")
+		return
+	else
+		M.adjustToxLoss(3 * REM)
```

Comment:
-3, come on, you *just* made the same mistake in the vox one

Reply:
No, I was at college and intended to fix it when I got home.

Reply:
Wait
This for *normal* people
So it would indeed be a +3

Reply:
And here I thought plasmemes would get *healed* by plasma.

Reply:
>/datum/reagent/plasma/on_mob_life(var/mob/living/M)
	if(..())
		return 1
	if(holder.has_reagent("inaprovaline"))
		holder.remove_reagent("inaprovaline", 2 * REM)
	M.adjustToxLoss(3 * REM)


I haven't touched the effects of Plasma on people.

Reply:
yeah i'm retarded ignore all this